src/pages/Record/Record.js

```diff
@@ -0,0 +1,236 @@
+
+    var chart = am4core.create(divName, am4charts.XYChart);
+
+    chart.colors.list = [am4core.color(color)];
+
+    chart.data = data;
+    chart.dateFormatter.inputDateFormat = 'yyyy-MM-dd';
+
+    var dateAxis = chart.xAxes.push(new am4charts.DateAxis());
+    dateAxis.renderer.labels.template.fill = am4core.color('#fff');
+    dateAxis.dateFormats.setKey('day', 'dd');
+    var valueAxis = chart.yAxes.push(new am4charts.ValueAxis());
+    valueAxis.renderer.labels.template.fill = am4core.color('#fff');
+
+    var series = chart.series.push(new am4charts.LineSeries());
+    series.dataFields.valueY = 'value';
+    series.dataFields.dateX = 'date';
+    series.tooltipText = '{value}' + unit;
+    series.strokeWidth = 2;
+    series.minBulletDistance = 15;
+
+    series.tooltip.background.cornerRadius = 20;
+    series.tooltip.background.strokeOpacity = 0;
+    series.tooltip.pointerOrientation = 'vertical';
+    series.tooltip.label.minWidth = 40;
+    series.tooltip.label.minHeight = 40;
+    series.tooltip.label.textAlign = 'middle';
+
+    const bullet = series.bullets.push(new am4charts.CircleBullet());
+    if (imageYn) {
+      const image = bullet.createChild(am4core.Image);
+      image.href = '/images/like_full.png';
+      image.width = 20;
+      image.height = 20;
+      image.horizontalCenter = 'middle';
+      image.verticalCenter = 'middle';
+    } else {
+      bullet.circle.strokeWidth = 2;
+      bullet.circle.radius = 4;
+      bullet.circle.fill = am4core.color('#fff');
+    }
+
+    chart.cursor = new am4charts.XYCursor();
+    chart.cursor.behavior = 'panXY';
+    chart.cursor.xAxis = dateAxis;
+    chart.cursor.snapToSeries = series;
+  };
+
+  return (
+    <div className="record contentsWrap">
+      <div className="titleWrap">
+        <h1 className="title">운동 기록</h1>
+      </div>
+
+      <div className="container">
+        <div className="myInfo">
+          <div className="subTitle">
+            <h2>오늘 내 기록</h2>
+          </div>
+          <div className="recordWrap">
+            <select name="workoutTime" onChange={e => handleRecord(e)}>
+              <option value="">운동시간</option>
+              <option value="0.5">30분</option>
+              <option value="1">1시간</option>
+              <option value="1.5">1시간30분</option>
+              <option value="2">2시간</option>
+              <option value="2.5">2시간30분</option>
+              <option value="3">3시간</option>
+              <option value="3.5">3시간30분</option>
+              <option value="4">4시간</option>
+              <option value="4.5">4시간30분</option>
+            </select>
+            <input
+              type="number"
+              className="inputBox"
+              placeholder="심박수"
+              onChange={e => handleRecord(e)}
+            />
+            <input
+              type="number"
+              className="inputBox"
+              placeholder="체중"
+              onChange={e => handleRecord(e)}
+            />
```

Comment:
- 최상위 요소(recordWrap)에서 onChange 이벤트를 한 번만 부착해도 하위 요소들을 전부 관리할 수 있을 것 같습니다. 이벤트 위임(+캡쳐링, 버블링)이라는 개념에 관련된 내용이니, [관련 아티클](https://joshua1988.github.io/web-development/javascript/event-propagation-delegation/) 반드시 읽어보세요! 면접에서 자주 물어보는 질문입니다.
```suggestion
          <div className="recordWrap" onChange={handleRecord}>
            <select name="workoutTime">
              <option value="">운동시간</option>
              <option value="0.5">30분</option>
              <option value="1">1시간</option>
              <option value="1.5">1시간30분</option>
              <option value="2">2시간</option>
              <option value="2.5">2시간30분</option>
              <option value="3">3시간</option>
              <option value="3.5">3시간30분</option>
              <option value="4">4시간</option>
              <option value="4.5">4시간30분</option>
            </select>
            <input
              type="number"
              className="inputBox"
              placeholder="심박수"
            />
            <input
              type="number"
              className="inputBox"
              placeholder="체중"
            />
```
- input에 name attribute를 부여해야 계산된 속성명으로 state를 업데이트할 수 있겠네요!